noir-projects/aztec-nr/aztec/src/context/public_context.nr

```diff
@@ -42,7 +42,6 @@ struct PublicContext {
     new_l2_to_l1_msgs: BoundedVec<L2ToL1Message, MAX_NEW_L2_TO_L1_MSGS_PER_CALL>,
 
     unencrypted_logs_hashes: BoundedVec<LogHash, MAX_UNENCRYPTED_LOGS_PER_CALL>,
-    unencrypted_log_preimages_length: Field,
```

Comment:
@ AVM team - removed total logs length from public context, as I believe it's not required for you.